### PR TITLE
KIALI-2727 [operator] extend service

### PR DESCRIPTION
**Pull request by jmazzitelli**
This follows up on https://github.com/kiali/kiali/pull/1013. Do NOT merge this PR until that PR 1013 is merged.

See: https://issues.jboss.org/browse/KIALI-2727

### CODE REVIEW

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#933](https://paste.fedoraproject.org/paste/hRQ8acdcMDeUQtS-Z3QagA)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1739]](https://paste.fedoraproject.org/paste/rrZHpPSzkjFKN7NBiXOLew)

**Comment by rhqci** [comment]
!!! Couldn't read commit file !!!

File: operator/deploy/kiali/kiali_cr.yaml [hunk 1]
```diff
@@ -4,6 +4,20 @@ metadata:
...
+###################################################################
+# kiali_cr.yaml
+#
+# This is a fully documented Kiali custom resource yaml file.
+# It can be used to tell the Kiali Operator to install Kiali.
+#
+# This is actually an empty Kiali CR, however, it provides
+# documentation on all available settings.
+# In each documented section, you will see a "---" marker;
+# below that marker you will see the name of the settings along
```
[the remark this answers is not in the document]
Comment (jmazzitelli) [reply operator/deploy/kiali/kiali_cr.yaml]: fixed.. need another approval

**Comment by rhqci** [comment]
Jenkins CI: kiali-core-pr-e2e-test [#936](https://paste.fedoraproject.org/paste/H3aoRPbl1tttci9T28RJtQ)
  * :heavy_check_mark: run-kiali-e2e-tests [#[1743]](https://paste.fedoraproject.org/paste/FBlOegS8QgKh~JC0iuKsmQ)

**Pull request merged**